frontend/src/components/GroupView/RightView/index.tsx

```diff
@@ -14,24 +26,42 @@ const EditGroupNameIcon = (): ReactElement => {
   return <SamwiseIcon iconName="pencil" className={styles.EditGroupNameIcon} onClick={handler} />;
 };
 
-const RightView = ({ group, groupMemberProfiles }: Props): ReactElement => (
-  <div className={styles.RightView}>
-    <div className={styles.GroupTaskCreator}>
-      <TaskCreator view="group" group={group.name} />
-    </div>
+export default class RightView extends React.PureComponent<Props, State> {
```

Comment:
Is there a reason why we have to use a class component instead of function component and with hooks

Reply:
It slipped my mind that React had a state hook, I can change `RightView` back to a functional class in a later commit.